src/events/2018.json

```diff
@@ -6,7 +6,7 @@
         "date": "Sábado - 05/05",
         "hour": "8:00 - 19:00",
         "place": "UFSCar - Anfiteatro Bento Prado",
-        "sponsor_link": ""
+        "sponsor_link": "Documento de Patrocínio - Devconf 2018.pdf"
```

Comment:
This should be a link 🤔 I think that miss a `/` here. At least good ideia is change the name to: `documento_de_patrocinio_devconf_2018.pdf`

Reply:
¬¬

Reply:
works fine locally